connectors/kafka/src/test/java/io/camunda/connector/kafka/inbound/KafkaExecutableTest.java

```diff
@@ -12,28 +12,17 @@
 import static org.junit.jupiter.api.Assertions.assertNotNull;
 import static org.mockito.ArgumentMatchers.any;
 import static org.mockito.ArgumentMatchers.argThat;
-import static org.mockito.Mockito.doAnswer;
-import static org.mockito.Mockito.doNothing;
-import static org.mockito.Mockito.timeout;
-import static org.mockito.Mockito.times;
-import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.*;
```

Comment:
Just a small thing: we have an agreement that we don't use wildcard imports. IntelliJ automatically uses wildcards but you can turn this off: https://www.jetbrains.com/help/idea/creating-and-optimizing-imports.html#disable-wildcard-imports

Reply:
@markfarkas-camunda Good point. I think it depends on the number of imports but if that has been decided, shouldnt we then reflect this in our spotless config?

Reply:
Fixed